app/src/components/Scheduling/ContactReview.js

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useState } from 'react';
+import Header from './Header';
+import {
+  TextField,
+  FormControl,
+  Button,
+} from '../../store/index';
+import { useStyles } from './ContactReviewStyles';
+
+const HARD_CODED_DATA = {
+  caseWorker: 'Tracy',
+  firstName: 'Nina',
+  lastName: 'Mosley',
+  email: 'nina99@gmail.com',
+  phoneNumber: '+1 (213) 555-5555'
+}
+
+export default function ContactReview() {
+  const classes = useStyles();
+  const [inputs, setInputs] = useState({
+    caseWorker: '',
+    firstName: '',
+    lastName: '',
+    email: '',
+    phoneNumber: '',
+  })
+
+  const [editMode, toggleEdit] = useState({
+    email: false,
+    phoneNumber: false,
+  });
+
+  useEffect(() => {
+    setInputs(HARD_CODED_DATA);
+  }, [])
+
+  const onChange = (e) => {
+    setInputs({...inputs, [e.target.name]: e.target.value})
+  };
+
+  return (
+    <div className={classes.root}>
+      <Header
+        title="Contact Information"
+        description="Review your profile information below"
+        activity={`Training with ${HARD_CODED_DATA.caseWorker}`}
+      />
+
+      <FormControl>
+        <>
+          <label className={classes.letterSpacing}>First Name<span className={classes.required}>*</span>
+          </label>
+          <TextField
```

Comment:
I spent a few minutes trying to figure out why these <inputs> get rendered with so much padding. In this mock, the inputs have a height of 48px: https://www.figma.com/file/kwUcXHYpKMHhuktdElLmmw/Calendaring-Home-Unite-Us-Copy?node-id=140%3A969

But in the screenshot, they're about 58px. I think it's because there's a lot of padding (circled in the screenshot), but I couldn't pinpoint where it's getting that padding from. I can look again tomorrow. 

<img width="1680" alt="Screen Shot 2021-05-02 at 22 25 59" src="https://user-images.githubusercontent.com/63125920/116843694-8f71aa80-ab95-11eb-94db-5e48e8552490.png">

Reply:
@Alvindelito Do you know where all this padding is coming from?

Reply:
I'm not entirely sure how you got that extra padding. There shouldn't be an extra input in that space.

Here is what it looks like on my screen when after a fresh reload:
![Screen Shot 2021-05-04 at 9 20 50 AM](https://user-images.githubusercontent.com/17840013/117036251-14ee7b00-acba-11eb-8397-f81dc5f1b7f7.png)

The extra space between `Training with Tracy`  and `First Name*` is coming from flexbox

Reply:
Actually, after taking a closer look, I can see an extra `<h6>` in the space between `Training with Tracy` and `First Name*`. This is coming from `<Header />` and the extra space should go away if I get the okay to remove the `children` prop in the other comment

Reply:
My bad, that extra input was there because I was experimenting with something. I'm talking about the padding in the input fields, like the one that contains `Nina`. They're much taller than the 48px height in the Figma link in this thread.

Reply:
Good find on the extra h6 though.

Reply:
Ahhh I see now. This is the default style of `<TextField>`. I should've looked at it closer to match the Figma style. I'll get that fixed shortly